cp3pt0-deployment/common/migrate_singleton.sh

```diff
@@ -81,6 +76,88 @@ function main() {
     success "Migration is completed for Cloud Pak 3.0 Foundational singleton services."
 }
 
+function migrate_license_service_reporter(){
+    title "LSR migration from ibm-cmmon-services to ${LSR_NAMESPACE}"
+
+    local lsr_instances=$("$OC" get IBMLicenseServiceReporter instance -n ibm-common-services --no-headers | wc -l)
```

Comment:
We shall not assume it's alays `ibm-common-services` - i.e. sometimes the customer / Pak could have configured the namespace into the custom namespace name

Reply:
changed to ${OPERATOR_NS}